app/assets/stylesheets/icons/icons.scss

```diff
@@ -182,6 +182,13 @@
         content: $icon-lp;
     }
 }
+.icon-coin,
+.blacklight-coin .document-thumbnail .default {
+    &:before {
+        content: $icon-coins-stack;
+        // content: $icon-coins-flat;
```

Comment:
This comment was not removed after being disabled.

Reply:
Fixed.